### DISCUSSION TITLE
Reply to an Incoming wa.id

### DISCUSSION THREAD

**Discussion by derrickobedgiu1** [netflie/whatsapp-cloud-api#79] [Ideas]
Could we have [context] parameters passed in the outgoing requests to include an option to reply to a message that just dropped in via the webhook? This can be an optional parameter.
A user could send multiple messages before the business can reply and reply to one that isn't the current one would be a challenge. Adding the [context] parameter could help with this.

**Comment by avido** [comment]
What's the latest status on this?

I've seen a PR (https://github.com/netflie/whatsapp-cloud-api/pull/122)  but seems abandoned.
